router_test.go

```diff
@@ -329,7 +329,12 @@ func Benchmark_Router_Chain(b *testing.B) {
 	handler := func(c *Ctx) {
 		c.Next()
 	}
-	app.Get("/", handler, handler, handler, handler, handler, handler)
+	app.Get("/", handler)
```

Comment:
please don´t change the benchmark, in this bench the handlers should be in one line

to measure the compression of the route we should create an independent benchmark, if that was the idea behind the change

Reply:
Ok I’ll add another one